virtualbox/create-env.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+set -eu
```

Comment:
should we add -x ?

Reply:
no; this should be as human-friendly as we can make it. if they're curious, they should look at the script